doc/quickstart/configure.rst

```diff
@@ -271,192 +274,173 @@ For example, Python's ``None`` is YAML's ``null``, Python's ``True`` is YAML's
 
 .. note::
 
   If not all preprocessor functions support lazy data, computational
-  performance may be best with the :ref:`default scheduler <config-dask-default-scheduler>`.
+  performance may be best with the :ref:`default scheduler
+  <config-dask-default-scheduler>`.
   See :issue:`674` for progress on making all preprocessor functions lazy.
 
-**Example configurations**
-
 *Personal computer*
 
-Create a Dask distributed cluster on the computer running ESMValCore using
-all available resources:
+Create a :class:`distributed.LocalCluster` on the computer running ESMValCore
+using all available resources:
 
 .. code:: yaml
 
-  cluster:
-    type: distributed.LocalCluster
+  dask:
+    run: local_cluster  # use "local_cluster" defined below
+    clusters:
+      local_cluster:
+        type: distributed.LocalCluster
 
-this should work well for most personal computers.
+This should work well for most personal computers.
 
 .. note::
 
-   Note that, if running this configuration on a shared node of an HPC cluster,
-   Dask will try and use as many resources it can find available, and this may
-   lead to overcrowding the node by a single user (you)!
+   If running this configuration on a shared node of an HPC cluster, Dask will
+   try and use as many resources it can find available, and this may lead to
+   overcrowding the node by a single user (you)!
 
 *Shared computer*
 
-Create a Dask distributed cluster on the computer running ESMValCore, with
-2 workers with 4 threads/4 GiB of memory each (8 GiB in total):
+Create a :class:`distributed.LocalCluster` on the computer running ESMValCore,
+with 2 workers with 2 threads/4 GiB of memory each (8 GiB in total):
 
 .. code:: yaml
 
-  cluster:
-    type: distributed.LocalCluster
-    n_workers: 2
-    threads_per_worker: 4
-    memory_limit: 4 GiB
+  dask:
+    run: local_cluster  # use "local_cluster" defined below
+    clusters:
+      local_cluster:
+        type: distributed.LocalCluster
+        n_workers: 2
+        threads_per_worker: 2
+        memory_limit: 4GiB
 
 this should work well for shared computers.
 
 *Computer cluster*
 
-Create a Dask distributed cluster on the
-`Levante <https://docs.dkrz.de/doc/levante/running-jobs/index.html>`_
-supercomputer using the `Dask-Jobqueue <https://jobqueue.dask.org/en/latest/>`_
-package:
+Create a Dask distributed cluster on the `Levante
+<https://docs.dkrz.de/doc/levante/running-jobs/index.html>`__ supercomputer
+using the `Dask-Jobqueue <https://jobqueue.dask.org/en/latest/>`__ package:
 
 .. code:: yaml
 
-  cluster:
-    type: dask_jobqueue.SLURMCluster
-    queue: shared
-    account: bk1088
-    cores: 8
-    memory: 7680MiB
-    processes: 2
-    interface: ib0
-    local_directory: "/scratch/b/b381141/dask-tmp"
-    n_workers: 24
+  dask:
+    run: slurm_cluster  # use "slurm_cluster" defined below
+    clusters:
+      slurm_cluster:
+        type: dask_jobqueue.SLURMCluster
+        queue: shared
+        account: bk1088
```

Comment:
I'd remove this as public info, Manu

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2616/commits/a40fb69124649952bdc45b535d6269429658f5ee